pageserve/static/style.css

```diff
@@ -3,6 +3,7 @@
   background-color: #333;
   overflow: hidden;
   display: flex;
+  flex-wrap: wrap;
```

Comment:
How will this look if the user is signed in? (since there will be an extra box)

Reply:
The second line is just a bit longer. flex-wrap automatically wrap flexbox elements to the next row if they don't fit on the current one.